### ISSUE TITLE
"$AbortOnUncleanConfig on" doesn't work

### ISSUE THREAD

**Issue by Whissi** [rsyslog/rsyslog#78]
1. Create ```/etc/rsyslog.conf``` with the following content:

  ```
  $AbortOnUncleanConfig on
  
  module(load="imuxsock")
  
  module(
  	load="builtin:omfile"
  	Template="RSYSLOG_TraditionalFileFormat"
  	FileCreateMode="0644"
  	DirCreateMode="0755"
  )
  
  *.* action(
  	type="omfile"
  	File="/var/log/test.log"
  	FileOwner="root"
  	FileGroup="adm"
  	anError="true"
  )
  ```

2. Now try to start ```rsyslogd```: ```/usr/sbin/rsyslogd -n -f /etc/rsyslog.conf```

3. rsyslog will report a configuration error
  ```
  error during parsing file /etc/rsyslog.conf, on or before line 18: parameter 'anError' not known -- typo in config file? [try http://www.rsyslog.com/e/2207 ]
  ```
but will continue to start

Expected result: rsyslogd should exit with an error due to ```$AbortOnUncleanConfig on```

...and BTW: You cannot quit ```rsyslogd``` with ```CTRL+C``` when running with ```-n``` without ```-d``` for example.

Tested with v7.6.3.

**Comment by rgerhards** [comment]
OK, finally coming to this issue. It requires more work than it sounds like. The problem is that the Bison parser does not longer offer the status information back. So a new status indicator needs to be added, and unfortunatley it needs to be maintained all over the code that does config processing. Not complicated, but lots of code to look at.

**Comment by rgerhards** [comment]
fix to be released with 8.10.0.
